src/sentry/issues/status_change.py

```diff
@@ -117,4 +119,11 @@ def handle_status_update(
                 kwargs={"project_id": group.project_id, "group_id": group.id}
             )
 
+        post_save.send(
```

Comment:
@wedamija are we not calling `group.save` anywhere?

Reply:
Not for when we call this function - we just use `.filter(...).update(...)`

Reply:
@wedamija can we override the `update` method too?

Reply:
Not really, because we don't know which rows were updated. This is doing an update directly in postgres, usually for efficiency purposes. I'm specifically talking about `.filter(...).update(...)` vs `Model.update` by the way, the latter does send a post save

Reply:
hmmm ok, can we audit all the times we call `update` on an issue?

Reply:
It's probably worth doing, I need to have a deeper look at how we're syncing anyway. This logic seems a little weird to me in general https://github.com/getsentry/sentry/blob/0132f2bf5b0219655234b3bbceaa84a9947d5f69/src/sentry/issues/attributes.py#L203-L214

Like if `.save()` is called without update fields we won't sync either...